core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -47,43 +74,69 @@
+                 ng-mouseover="updateExplorationPreview(node.getExplorationId())"
+                 ng-mouseleave="togglePreviewCard()"
+                 class="oppia-dashboard-list-summary">
+                <[node._explorationSummaryObject.objective]>
+              </a>
+            </td>
+          </tr>
+        </table>
 
-        <div ng-if="showingAllExplorations && (getNonRecommendedCollectionNodeCount() > 0)" class="oppia-collection-player-tile-section oppia-collection-animate-slide">
-          <div class="oppia-collection-player-small-text">
-            Upcoming exploration<span ng-if="getNonRecommendedCollectionNodeCount() > 1">s</span> in this collection
-          </div>
-          <collection-node-list collection-id="collectionId" collection-nodes="getNonRecommendedCollectionNodes()"></collection-node-list>
-        </div>
 
-        <a class="oppia-collection-player-small-text" ng-if="getNonRecommendedCollectionNodeCount() > 0" ng-click="toggleShowAllExplorations()">
-          <span ng-if="!showingAllExplorations"><span class="fa fa-chevron-down"></span> Show upcoming explorations</span>
-          <span ng-if="showingAllExplorations"><span class="fa fa-chevron-up"></span> Only show current progress</span>
-        </a>
-      {% else %}
-        <div class="oppia-collection-player-small-text">You must login to save your progress.</div>
-        <collection-node-list collection-id="collectionId" collection-nodes="collection.getCollectionNodes()"></collection-node-list>
-      {% endif %}
+        <div ng-if="collection" class="card-preview-panel hidden-xs">
+
+          <md-card class="oppia-activity-summary-tile md-default-theme">
+            <div class="title-section" style="background-color: <[collectionSummary.thumbnail_bg_color]>; z-index: 1;">
+              <img ng-src="<[collectionSummary.thumbnail_icon_url]>" class="thumbnail-image">
+            </div>
+          </md-card>
+
+        <exploration-summary-tile ng-if="!previewCardIsShown"
+                                  collection-id="getCollectionId()"
+                                  exploration-id="currentExplorationId"
+                                  exploration-title="summaryToPreview.title"
+                                  last-updated-msec="summaryToPreview.last_updated_msec"
+                                  objective="summaryToPreview.objective"
+                                  category="summaryToPreview.category"
+                                  ratings="summaryToPreview.ratings"
+                                  num-views="summaryToPreview.num_views"
+                                  thumbnail-icon-url="summaryToPreview.thumbnail_icon_url"
+                                  thumbnail-bg-color="summaryToPreview.thumbnail_bg_color"
+                                  is-community-owned="summaryToPreview.community_owned"
+                                  style="position: absolute; left: 75px; z-index: 10;"/>
```

Comment:
Don't use `/>`; close with an explicit `</exploration-summary-tile>` tag. I don't think our custom tags are meant to be self-closing.